contrib/packs/actions/pack_mgmt/download.py

```diff
@@ -26,141 +25,94 @@
 from st2actions.runners.pythonrunner import Action
 from st2common.util.green import shell
 
-ALL_PACKS = '*'
-PACK_REPO_ROOT = 'packs'
 MANIFEST_FILE = 'pack.yaml'
 CONFIG_FILE = 'config.yaml'
 GITINFO_FILE = '.gitinfo'
 PACK_RESERVE_CHARACTER = '.'
-
-STACKSTORM_CONTRIB_REPOS = [
-    'st2contrib',
-    'st2incubator'
-]
-
-#####
-# !!!!!!!!!!!!!!
-# !!! README !!!
-# !!!!!!!!!!!!!!
-#
-# This NEEDS a rewrite. Too many features and far too many assumption
-# to keep this impl straight any longer. If you only want to read code do
-# so at your own peril.
-#
-# If you are here to fix a bug or add a feature answer these questions -
-# 1. Am I fixing a broken feature?
-# 2. Is this the only module in which to fix the bug?
-# 3. Am I sure this is a bug fix and not a feature?
-#
-# Only if you can emphatically answer 'YES' to allow about questions you should
-# touch this file. Else, be warned you might loose a part of you soul or sanity.
-#####
-
+PACK_VERSION_SEPARATOR = '#'
```

Comment:
No one use `=` for this purpose as it seems. NPM uses `#` and PIP sticks to `@`. I like `#` more as it correlates with anchor part of the URL and also doesn't repeat throughout the rest of the url so you can just split by character.